react/src/components/buttons/SprkButton.stories.js

```diff
@@ -134,9 +134,41 @@ export const spinningTertiary = () => (
   </SprkButton>
 );
 
-export const fullWidthAtSmallViewport = () => (
+export const compact = () => (
+  <SprkButton
```

Comment:
Let's go through and update all the button stories in here to be in order. There's no `button-4` or `button-8` so let's just update everything and have the numbers match on the `idString` and the `analyticsString` for each button.